src/m365/aad/commands/o365group/o365group-add.ts

```diff
@@ -179,6 +138,50 @@ class AadO365GroupAddCommand extends GraphCommand {
       }, (rawRes: any): void => this.handleRejectedODataJsonPromise(rawRes, logger, cb));
   }
 
+  private getUserIds(logger: Logger, users: string | undefined): Promise<string[]> {
+    if (!users) {
+      if (this.debug) {
+        logger.logToStderr('No users to validate, skipping.');
+      }
+      return Promise.resolve([]);
+    }
+
+    if (this.verbose) {
+      logger.logToStderr('Retrieving user information.');
+    }
+
+    const userArr: string[] = users.split(',').map(o => o.trim());
+    let promises: Promise<{ value: { id: string; userPrincipalName: string }[] }>[] = [];
+    let userIds: string[] = [];
+
+    promises = userArr.map(user => {
+      const requestOptions: any = {
+        url: `${this.resource}/v1.0/users?$filter=userPrincipalName eq '${encodeURIComponent(user).replace(/'/g, `''`)}'&$select=id,userPrincipalName`,
```

Comment:
We can replace `encodeURIComponent(user).replace(/'/g, `''`)` with `Utils.encodeQueryParameter(string)`